build/runServices.sh

```diff
@@ -10,6 +10,10 @@ for svc in ${svcs[@]}; do
   echo "${svc} --registry_address=etcd:2379 --server_address=device-manager:${port}"
   ${svc} --registry_address=etcd:2379 --server_address=device-manager:${port} >> /var/log/deviceManager/${svc}.log 2>&1 &
   ((port=port+1))
+  if [ $svc == 'svc-task' ]
+  then
+    ((port=port+1))
```

Comment:
Why are we incrementing port for svc-task twice?

Reply:
I wanted for ports to be consistent with those in ODIM, so there is no double usage (in ODIM svc-manager is using 45107 and svc-update is using 45108). But I checked, and port 45106 is not being used, so the second incrementation is not necessary.